front/app/components/EsriInitiativesMap/components/InitiativePreview.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+
+// components
+import Button from 'components/UI/Button';
+import {
+  Icon,
+  Text,
+  colors,
+  Title,
+  Box,
+} from '@citizenlab/cl2-component-library';
+import Body from 'components/PostShowComponents/Body';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from '../messages';
+
+// style
+import styled from 'styled-components';
+
+// hooks
+import useInitiativeById from 'api/initiatives/useInitiativeById';
+import useLocalize from 'hooks/useLocalize';
+
+// utils
+import clHistory from 'utils/cl-router/history';
+import { updateSearchParams } from 'utils/cl-router/updateSearchParams';
+
+const Container = styled.div`
+  flex: 1;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  padding: 30px;
+  position: relative;
+  overflow: hidden;
+`;
+
+const DescriptionPreview = styled.div`
```

Comment:
These styled components are what we had from before. I've just re-used them since I don't think we have a better way to do this existing opacity gradient style on the description text.